tests/test_utils.py

```diff
@@ -0,0 +1,56 @@
+from Crypto.PublicKey import RSA 
+from Crypto.Signature import pkcs1_15 
+from Crypto.Hash import SHA256 
+import pytest
+from pcdcutils.gen3 import Gen3RequestManager
+
+# openssl genpkey -algorithm RSA -out PRIVATE_NAME.pem -pkeyopt rsa_keygen_bits:2048
+# openssl rsa -pubout -in PRIVATE_NAME.pem -out PUB_NAME.pem
+# To run the tests, create a keys directory in the tests folder and create key pairs private/public_key1 and 2.pem
+def test_successful_make_sig():
+  body = 'aaaaa'
+  pri_key_path = '/Users/ritroy/pcdcutils/tests/keys/private_key1.pem'
```

Comment:
Add the keys in a tests/data folder and reference the one here for testing otherwise this will all fail

Reply:
@microbe73 you need to update the path, `/Users/ritroy/` is the user on your machine and it won't work for anyone else.